src/terra/Command/Environment/EnvironmentEnable.php

```diff
@@ -70,7 +70,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         // Write drush alias.
-        $drush_alias_file_path = "{$_SERVER['HOME']}/.drush/{$app_name}.aliases.drushrc.php";
+        $drush_alias_file_path = "{$_SERVER['HOME']}/.drush/terra.{$app_name}.aliases.drushrc.php";
```

Comment:
Yes. sorry, this line is technically unrelated to this PR, but it is a bug. I can pull it out into a separate PR, if you want.